helpers/theme.js

```diff
@@ -6,14 +6,40 @@ const {
 	HOST,
 } = require('../config/global');
 
+const instanceSpecificTitle = () => {
+	switch (SC_THEME) {
+		case 'n21':
+			return 'Niedersächsischen Bildungscloud';
+		default:
+			return SC_TITLE;
+	}
+};
+
+const instanceInstitute = () => {
+	switch (SC_THEME) {
+		case 'n21':
+			return 'Landesinitiative n-21: Schulen in Niedersachsen online e.V.';
```

Comment:
hier siehe Kommentar zum Commit, Punkt hinter "e.V" sollte weggelassen werden bedingt durch Einsatz in den Textabschnitt, da sonst zwei Punkte hintereinander